content/docs/command-reference/dag.md

```diff
@@ -20,7 +20,7 @@ A data pipeline, in general, is a series of data processing
 input and produce an <abbr>output</abbr>). A pipeline may produce intermediate
 data, and has a final result.
 
-Data processing or ML pipelines typically start with large raw datasets, include
+Data science and ML pipelines typically start with large raw datasets, include
```

Comment:
Expand "ML"? This doc doesn't have it spelled out anywhere, it seems.

Reply:
I added the full term on lines 83-84. I agree it is better up here and simplifies the later example though. Made that change.